packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1087,13 +1092,20 @@ class ThemeData with Diagnosticable {
   /// start using new colors, typography and other features of Material 3.
   /// If false, they will use the Material 2 look and feel.
   ///
+  /// If true, the default Typography will be [Typography.material2021],
+  /// otherwise it will default to [Typography.material2014].
+  /// 
   /// If a [ThemeData] is constructed with [useMaterial3] set to true,
   /// the default [typography] will be [Typography.material2021],
   /// otherwise it will be [Typography.material2014].
   ///
   /// However, just copying a [ThemeData] with [useMaterial3] set to true will
   /// not change the typography of the resulting ThemeData.
   ///
+  /// If true, the default [splashFactory] will be [InkSparkle] if and only if
+  /// the platform is Android and the app is compiled to run on the web.
+  /// Otherwise, it will default to [InkSplash].
+  /// 
```

Comment:
Perhaps we could combine this with the typography paragraph above as they have a similar effect. They only apply to the constructor and not copyWith.

Reply:
Ok I refactoring this paragraph, I think it will help as future things are added under this flag

Reply:
LMK if this new structure works. When a new property is added, a list item an be added

Reply:
Thanks @guidezpl for re-formatting with nice markdown